### PR TITLE
[W5.1a][W13-2]Arjo Chakravarty

### PR DESCRIPTION
**Pull request by arjo129**
Completed LO-SRP

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+
+    public void showInitFailedMessage() {
+        showToUser(MESSAGE_INIT_FAILED, DIVIDER, DIVIDER);
+    }
+
+    /** Shows message(s) to the user */
+    public void showToUser(String... message) {
+        for (String m : message) {
+            out.println(LINE_PREFIX + m.replace("\n", LS + LINE_PREFIX));
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:60]: Here, the `Formatter` is also doing output, and thus violates SRP.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -21,34 +21,22 @@
...
     /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
     private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
 
     private final Scanner in;
-    private final PrintStream out;
 
-    public TextUi() {
-        this(System.in, System.out);
-    }
+    private static final String LINE_PREFIX = "|| ";
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/TextUi.java:31]: `LINE_PREFIX` can go to the formatter, no?

**Review by okkhoy — COMMENTED** [review]
It looks like you may be confused about SRP. Please take a look at the relevant content in the text.
For this particular exercise:
You can think of doing all formatting (like appending prefixes, line separators, trimming etc.,) in the Formatter.

**Comment by okkhoy** [comment]
@arjo129 
Some comments added. Please take note and close the pull request.

**Comment by arjo129** [comment]
Noted. Thank you.

**Pull request closed without merging**